glowing_waffles/notebooks/acp.py

```diff
@@ -0,0 +1,107 @@
+class ObserveACP:
+    
+    def __init__(self, name, date):
+        
+        from astropy.coordinates import SkyCoord
+        from astropy.time import Time
+        import datetime
+    
+        self.name = name
+        self.filters = []
+        self.binning = 1
+        
+        self.ra = SkyCoord.from_name(name).ra.deg
+        self.dec = SkyCoord.from_name(name).dec.deg
```

Comment:
You can make this ~2x faster by using an intermediate variable to store the SkyCoord, then get the ra and dec from that:

```python
coord = SkyCoord.from_name(name)
self.ra = coord.ra.deg
```
That way the look-up is only done once.